.github/workflows/platformDeploy.yml

```diff
@@ -165,7 +165,7 @@ jobs:
 
       - name: Build staging desktop app
         if: ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
-        run: npm run desktop-build-staging -- --publish always
```

Comment:
FYI - notarize/electron will look at env variables to determine if we should notarize or not.